pandas/tests/frame/test_combine_concat.py

```diff
@@ -347,6 +347,20 @@ def test_concat_named_keys(self):
                                              names=[None, None]))
         assert_frame_equal(concatted_unnamed, expected_unnamed)
 
+    def test_concat_axis_parameter(self):
+        # GH 14369
+        df1 = pd.DataFrame({'A': [0.1, 0.2]}, index=range(2))
+        df2 = pd.DataFrame({'A': [0.3, 0.4]}, index=range(2))
+        expected_row = pd.DataFrame(
+            {'A': [0.1, 0.2, 0.3, 0.4]}, index=[0, 1, 0, 1])
+        concatted_row = pd.concat([df1, df2], axis='rows')
```

Comment:
What does this mean?  I looked through the `concat` parameters (as well as some other tests) and didn't see `index=0`.  Do you mean joining dataframes without indices?